extensions/sql-database-projects/src/dialogs/deployDatabaseDialog.ts

```diff
@@ -161,10 +161,10 @@ export class DeployDatabaseDialog {
 			sqlCmdVariables: this.project.sqlCmdVariables
 		};
 
+		azdata.window.closeDialog(this.dialog);
 		await this.deploy!(this.project, profile);
```

Comment:
do we show the in-progress indicator during the deployment? maybe we should leverage the task to do the deployment without letting the dialog to wait for the operation to complete.